scripts/uberenv/packages/axom/package.py

```diff
@@ -0,0 +1,459 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
How is this different from what's in the spack repo?

Reply:
The fact that this one exists and my PR is not approved.

Reply:
Ahh, so an axom package doesn't exist yet in spack? Got it.